bql/semantic/expression.go

```diff
@@ -249,6 +249,40 @@ func (e *comparisonForNodeLiteral) Evaluate(r table.Row) (bool, error) {
 	}
 }
 
+// comparisonForPredicateLiteral represents the internal representation of an expression of comparison between a binding and a predicate literal.
+type comparisonForPredicateLiteral struct {
+	op OP // operation.
+
+	lB  string // left binding.
+	rPL string // right predicate literal.
+}
+
+func (e *comparisonForPredicateLiteral) Evaluate(r table.Row) (bool, error) {
+	eL, err := cellFromRow(e.lB, r)
```

Comment:
nit/optional: `el` is ok here, you don't need to capitalize the `L` just because it is short for literal xD

Reply:
Good to know, I thought Go was more inflexible with regards to camelCase in this scenario.
I am refactoring this `expression.go` file in the PR 139, mainly to improve some variable names, so for now I will just let it as it is as it will already be changed there.